.kokoro/cleanup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -xeo pipefail
+
+export NPM_CONFIG_PREFIX=/home/node/.npm-global
+
+# Setup service account credentials.
+export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/spanner-key.json
+export GCLOUD_PROJECT=long-door-651
+
+cd $(dirname $0)/..
+
+npm install
+
+npm run cleanup
```

Comment:
How often / when is cleanup.sh run? It would be nice if there was a way to run it from the system tests, if we get an error about too many instances existing.

Reply:
It's run every hour